src/vm/interpreter.cpp

```diff
@@ -10316,15 +10321,19 @@ void Interpreter::CallI()
             }
             else
             {
-                pMD = g_pPrepareConstrainedRegionsMethod;  // A random static method.
+                pMD = g_pExecuteBackoutCodeHelperMethod;  // A random static method.
             }
             MethodDescCallSite mdcs(pMD, &mSig, ftnPtr);
+#if 0
```

Comment:
It triggers assertion in the Thread::ObjectRefFlush function at [threads.cpp:6861](https://github.com/dotnet/coreclr/blob/master/src/vm/threads.cpp#L6861):
```c++
 _ASSERTE(thread->PreemptiveGCDisabled());  // Should have been in managed code
```
I don't know much about GC subsystem and preemption in particular, so I just disabled that code.